iris/tests/awesomebar/alt_up_down_keys_in_awesomebar.py

```diff
@@ -14,7 +14,7 @@ def __init__(self):
         self.test_case_id = '108278'
         self.test_suite_id = '1902'
         self.exclude = Platform.LINUX
```

Comment:
we can remove self.exclude if the blocked by field has the same platform

Reply:
Unnecessary self.exclude flags have been removed.

Thank You